test/common_utils.py

```diff
@@ -406,26 +406,21 @@ def make_bounding_boxes(
     canvas_size=DEFAULT_SIZE,
     *,
     format=datapoints.BoundingBoxFormat.XYXY,
-    batch_dims=(),
+    num_objects=1,
```

Comment:
wait we're not setting this anywhere, right? Can we just add this parameter when / if we need it?